### ISSUE TITLE
`References` key not working in page YAML

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire#654]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

Adding `references` (as in example below) to page YAML does not add page-level biblio.

```
title: Chapter Title
order: 10
layout: page
references:
 - "West 1939"
 - "Evans 1938"
 - "Lynd 1929"
 ```

### Actual behavior

Nothing happens when a `references` key is added to the page YAML. The biblio does not appear. It doesn't break the preview. There is no messaging output related to `references` in the Terminal. 

### Expected behavior

Per v0, the `references` key should either add a page-level bibliography in place of what is created by the `{% cite %}` shortcode or it should add additional references to an existing bibliography (created by the `{% cite %}` shortcode) and sort accordingly.

PLEASE NOTE: While this key was originally called `references` in v0, we would like to suggest renaming it `citations` as this will connect it more to the `{% cite %}` shortcode (ie., there are two ways to add citations, with the `{% cite %}` shortcode or in the page yaml with `citations`). And anything called "references" should be a list like what we find in `references.yaml`.

### Version numbers

quire: 1.0.0-rc.0
node: 18.12.1

### What browsers are you seeing the problem on?

Chrome

### Supporting Information

This is related to GH Discussion #555 and [Quire-Docs Disccussion #440](https://github.com/thegetty/quire-docs/issues/440).

**Comment by Erin-Cecele** [comment]
Related to Jira issue [DEV-12066](https://jira.getty.edu/browse/DEV-12066)

**Comment by Erin-Cecele** [comment]
Just tested this, and it still isn't working. Needs further discussion with the core team.

**Comment by Erin-Cecele** [comment]
After discussion with the team, we opened a ticket to address the issue. This is a lower priority so will remain on backlog for the time being. In the meantime, if someone from the community is interested in addressing this issue, let us know!

**Comment by Erin-Cecele** [comment]
This issue was addressed in PR #818.
